v5/api/c/gps.rst

```diff
@@ -0,0 +1,681 @@
+.. highlight:: c
+   :linenothreshold: 5
+
+=========
+GPS C API
+=========
+
+.. note:: For a pros-specific usage guide on the GPS, please check out our article
+          `here <..\tutorials\topical\gps.rst>`_.
+
+.. note::
+   For the C++ API, check out it `here<../cpp/gps.rst>`_.
+
+.. contents:: :local:
+
+Functions
+=========
+
+gps_initialize_full
+-------------------
+
+Set the GPS's offset relative to the center of turning in meters, as well as its initial position.
+
+This function uses the following values of ``errno`` when an error state is reached:
+
+- ``ENXIO`` - The given value is not within the range of V5 ports (1-21).
+- ``ENODEV`` - The port cannot be configured as a GPS.
+- ``EAGAIN``` - The sensor is still calibrating.
+
+.. tabs ::
+   .. tab :: Prototype
+      .. highlight:: c
+      ::
+
+         int32_t gps_initialize_full(uint8_t port, double xInitial, double yInitial, double headingInitial, double xOffset,
```

Comment:
Prototype is not getting syntax highlighting:
<img width="959" alt="Screen Shot 2021-08-14 at 2 23 14 PM" src="https://user-images.githubusercontent.com/22580992/129456746-76a06b0a-4771-4fa7-8190-b5f4e21fd535.png">

Reply:
Gonna be honest, I don't see how it isn't getting proper syntax highlighting in the example above. It seems consistent with the rest of the docs.